common/djangoapps/third_party_auth/config/waffle.py

```diff
@@ -18,8 +17,21 @@
 # .. toggle_target_removal_date: 2021-04-31
 # .. toggle_warnings: None.
```

Comment:
I haven't gotten to review yet, but do you mind deleting `toggle_warnings` from this older annotation? Also, I think the second line of the `toggle_description` here will need more indents to not fail later.  Thank you!